src/lib/platform/EiScreen.cpp

```diff
@@ -695,10 +695,18 @@ void EiScreen::on_motion_event(ei_event* event)
          }
 #endif
     } else {
-        LOG_DEBUG("on_motion_event on secondary at (dx,dy)=(%.2f,%.2f)", dx, dy);
-        send_event(EventType::PRIMARY_SCREEN_MOTION_ON_SECONDARY,
-                   create_event_data<MotionInfo>(MotionInfo{static_cast<std::int32_t>(dx),
-                                                            static_cast<std::int32_t>(dy)}));
+        buffer_dx += dx;
+        buffer_dy += dy;
+        auto pixel_dx = (int32_t) buffer_dx;
+        auto pixel_dy = (int32_t) buffer_dy;
```

Comment:
```suggestion
        auto pixel_dx = static_cast<std::int32_t>(buffer_dx);
        auto pixel_dy = static_cast<std::int32_t>(buffer_dy);
```
I think, my C++ isn't great though.